### PR TITLE
Patient-doc-upload-frontend changes for the frontend

### PR DESCRIPTION
**Pull request by alpit4**

### CODE REVIEW

**Comment by alpit4** [comment]
1) In Desktop : -
Screenshot (236)
2) In Mobile:-
Mobile version

File: src/components/PatientList.js [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+  const [activePatientCount, setActivePatientCount] = useState(0);
+
+  useEffect(() => {
+    fetchData();
+  }, []);
+
+  const fetchData = async () => {
+    try {
+      const response = await axios.get(
+        "/openmrs/ws/rest/v1/bahmnicore/sql?location_uuid=833d0c66-e29a-4d31-ac13-ca9050d1bfa9&q=emrapi.sqlSearch.activePatients",
```
Comment (gsluthra) [inline src/components/PatientList.js]: Hardcoded location UUID for now?

**Comment by alpit4** [comment]
I have to hardcode the UUID from this file only or from all files .

File: src/components/PatientList.js (hunk 1 above)
Comment (gsluthra) [reply src/components/PatientList.js]: The Location UUID is a system generated ID for a location, and it is not going to be the same for every bahmni instance / location. Are you going to play a different story later, to read the actual UUID of a location via API?

Comment (alpit4) [reply src/components/PatientList.js]: Ok, I will hardcode it with the (location_uuid). So that anyone using it can put according to their location.

Comment (gsluthra) [reply src/components/PatientList.js]: The code needs to detect the location without any hardcoded location UUID. Can you do that?

Comment (alpit4) [reply src/components/PatientList.js]: I can try that

**Review by gsluthra — APPROVED** [review]

**Pull request merged**